web/package.json

```
@@ -10,7 +10,9 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint",
-    "test": "tsc --noEmit"
+    "test": "tsc --noEmit",
+    "test:unit": "vitest run",
```

Biome (2.4.15)

[error] The key test:unit was already declared.

(lint/suspicious/noDuplicateObjectKeys)